package.json

```diff
@@ -58,14 +59,7 @@
   "homepage": "https://github.com/trussworks/react-uswds#readme",
   "peerDependencies": {
     "react": "^16.x",
-    "react-dom": "^16.x"
-  },
-  "dependencies": {
-    "classnames": "^2.2.6",
-    "dotenv": "^8.2.0",
-    "happo-plugin-storybook": "^2.5.3",
-    "react": "^16.10.0",
-    "react-dom": "^16.10.0",
+    "react-dom": "^16.x",
     "uswds": "2.8.0"
```

Comment:
Do we want this peer dependency to specify exactly `2.8.0`, or do we want to allow consumers who might be on more current version of uswds to also use react-uswds without warnings?